### DISCUSSION TITLE
Sterocalibration for nonbinoclar cameras

### DISCUSSION THREAD

**Discussion by phochstrasser** [decadenza/SimpleStereo#33] [Q&A]
I'm pretty new to computer vision and I find only contradicting information and hope someone here can help me. Does stereoCalibrate from opencv only work for "real"/binocular stereo settings, or does stereo mean any camera pair with an overlapping FoV?
Also, could this library help me with this? thanks in advance!

**Comment by decadenza** [comment]
Hi @phochstrasser, and welcome.

Indeed it is hard to find a clear explanation of the basics behind stereo vision (l think lately the attention has been on machine learning methods for stereo matching, many take calibration and rectification for granted IMHO).

Not sure what you mean by "real" binocular images.
StereoCalibrate in OpenCV is used to calculate intrinsic and extrinsic parameters of a pair of cameras by using calibration images. Calibration images must represent the same object (commonly, a chessboard) at the same time from both cameras. This information is used by calibration algorithm(s).

This library also uses OpenCV, but at a slightly higher level.
I'd suggest to start from the examples provided here and see for yourself. You'll find a calibration one too.

Good luck. Open an issue or a discussion here when needed, happy to help.
